HARK/parallel.py

```diff
@@ -81,22 +83,24 @@ def multiThreadCommands(agent_list,command_list,num_jobs=None):
     None
     '''
     if len(agent_list) == 1:
-        multiThreadCommandsFake(agent_list,command_list)
+        multiThreadCommandsFake(agent_list, command_list)
         return None
 
     # Default number of parallel jobs is the smaller of number of AgentTypes in
     # the input and the number of available cores.
     if num_jobs is None:
-        num_jobs = min(len(agent_list),multiprocessing.cpu_count())
+        num_jobs = min(len(agent_list), multiprocessing.cpu_count())
 
     # Send each command in command_list to each of the types in agent_list to be run
-    agent_list_out = Parallel(backend='multiprocessing',n_jobs=num_jobs)(delayed(runCommands)(*args) for args in zip(agent_list, len(agent_list)*[command_list]))
+    delayed_commands = [delayed(runCommands)(*args) for args in zip(agent_list, len(agent_list)*[command_list])]
```

Comment:
This is where Ingrid and I rewrote the code a little bit - it was the only way to keep from breaking the line length limits.  I'm not 100% sure I didn't change the meaning of the code though.  Your original syntax here is very novel to me.

Reply:
If no new line was added beyond that, then this won't do anything.  It will just create a list of stuff to do, and then never do it.

Reply:
@mnwhite just to make sure - are you seeing line 96, with the code `agent_list_out = Parallel(backend='multiprocessing', n_jobs=num_jobs)(delayed_commands)`?  My understanding is that this runs the delayed commands, but you understand the code better than I.